packages/lesswrong/components/posts/PostsPage/ConvertedFromCommentMessage.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { registerComponent } from 'meteor/vulcan:core';
+import { withStyles } from '@material-ui/core/styles'
+import { Link } from '../../../lib/reactRouterWrapper.js';
+import { Comments } from '../../../lib/collections/comments';
+import { styles as messageStyles } from '../AlignmentCrosspostMessage';
+
+const styles = theme => ({
+  root: {
+    ...messageStyles(theme).root
+  },
+  link: {
+    color: theme.palette.primary.main
+  }
+});
+
+const ConvertedFromCommentMessage = ({post, classes }) => {
```

Comment:
I was thinking this would be edited into the text of the draft you create, so that you could modify or delete it. As it stands, having a "This post was originally a comment" message that you can't delete is a pretty strong incentive to create a post and copy-paste, rather than use this feature.

Reply:
Hmm. Good point.

Reply:
Do we have existing code that dynamically edits a post?

Reply:
I don't think so, and unfortunately this is pretty tricky because it needs to handle each editor type.